### PR TITLE
add timml

### PR DESCRIPTION
**Pull request by jentjr**

### CODE REVIEW

**Comment by jentjr** [comment]
@conda-forge/staged-recipes I believe everything is ready to go, and all checks have passed. Thanks in advance for reviewing.

File: recipes/timml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+about:
+  home: https://github.com/mbakker7/timml
+  license: MIT
+  license_file: LICENSE
+  summary: 'TimML is a multi-layer analytic element model'
+  dev_url: https://github.com/mbakker7/timml
+  
+extra:
+  recipe-maintainers:
+    - mbakker7
```
Comment (scopatz) [inline recipes/timml/meta.yaml]: @mbakker7 - are you OK with being a maintainer here?

Comment (mbakker7) [reply recipes/timml/meta.yaml]: Yes, absolutely. I am very glad @jentjr submitted this and I will happily be the maintainer.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**